src/program/lwaftr/doc/CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Change Log
 
+## [2018.06.01]
+
+### Features
+
+* Support influxdb format for snabb config by passing `--format influxdb`.
+
+* Support for large binding tables of above 4GB
+
+* Support TAP interface for the lwAFTR.
+
+* New snabb top with interactive interface (this replaces snabbvmx top too).
+    * Supports new interface view for condensed information per NIC.
+    * Support going back in time to see historical data.
+
+    See more information here:
+    https://github.com/Igalia/snabb/tree/lwaftr/src/program/top
+
+* Support running on RaptorJIT
```

Comment:
Ok, I understand this bullet as the lwAFTR is capable of running on RaptorJIT and we've made changes to make this possible. OTOH, I think it could be understood as the release runs on RaptorJIT. Maybe some extra wording is necessary to clarify this.